### PR TITLE
[Console command] System requirements check

### PR DESCRIPTION
**Pull request by mloock**
For CD/CI purposes, this PR is intented to provide a new console command to check if all the system requirements are satisfied. This is the same process as the one in the dedicated page in the UI.

image

There is also a special option to report only warning and/or errors.

image

This also provide a new static method to perform all the checks in one line (ready for some more future checks if necessary).

### CODE REVIEW

**Comment by brusch** [comment]
Very nice, thanks a lot!

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md [hunk 1]
```diff
@@ -125,10 +125,19 @@ Install facedetect for [image thumbnails focal point](../../04_Assets/03_Working
...
 
 Install html2text for [Pimcore mail](../../19_Development_Tools_and_Details/25_Email_Framework/01_Pimcore_Mail.md)
 
 ```bash
 apt-get install html2text
 ```
+
+# Check your installation
+
+You can check if everything is here in the UI, via the `Tools` / `System Info & Tools` / `System-Requirements Check` page.
```
Comment (dvesh3) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md:138]: You can check system requirements, via Admin UI `Tools` / `System Info & Tools` / `System-Requirements Check` menu. 

Or via following CLI command:

File: doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md [hunk 2]
```diff
@@ -124,3 +124,21 @@ Install facedetect for [image thumbnails focal point](../../04_Assets/03_Working
 ```bash
 apt-get install facedetect
 ```
+
+## html2text
```
Comment (dvesh3) [inline doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md:128]: not sure if merge conflicts resolved correctly. html2text section should be removed from here. please see https://github.com/pimcore/pimcore/pull/6448/files#diff-8c29cd6e0351c1a7c57ad1524fd58eaf

Comment (mloock) [reply doc/Development_Documentation/23_Installation_and_Upgrade/03_System_Setup_and_Hosting/06_Additional_Tools_Installation.md]: fixed! ;)

**Pull request merged**